### DISCUSSION TITLE
Why is the issue #419 marked as closed ? (RG351V: Missing on-screen keyboard for game meta data)

### DISCUSSION THREAD

**Discussion by mderie** [351ELEC/351ELEC#930] [Known Issues & Solutions]
Hi,
I had the same issue with the RG351M when entering the wifi password.
I had to edit the file under Linux on my laptop...
Regards

**Comment by medeirosT** [comment]
To our knowledge there is no issue with the virtual keyboard. What steps have you tried to open it?

**Comment by mderie** [comment]
When trying to edit the password for the wifi, the virtual keyboard doesn't appear. So it is impossible to enter edit mode.
The same goes when searching a game by name. I used the latest stable version from June 2022
See a screen capture here : https://github.com/AmberELEC/AmberELEC/issues/419

**Comment by dhwz** [comment]
The virtual keybord is working I just updated a really old image on my RG351V to the latest pre-release and the keyboard is still working.
Plese check "system settings -> developer > ui -> on-screen keyboard" is enabled, else of course it's not visible. But that setting is default on.

**Comment by mderie** [comment]
OMG ! I sincerely apologize...

It is working as you describe, so indeed everything is fine.

Thx a lot for pointing me in the right direction

**Comment by enfermero-dev** [comment]
Virtual keyboard on PPSSPP emulator won't show (anyway is mainly used just to change the user name on the emulator)
